lib/modules/platform/azure/util.ts

```diff
@@ -9,6 +9,10 @@ import { HostRule, PrState } from '../../../types';
 import type { GitOptions } from '../../../types/git';
 import { addSecretForSanitizing } from '../../../util/sanitize';
 import { toBase64 } from '../../../util/string';
+import {
+  hashBody,
+  isRebaseRequested,
+} from '../../../workers/repository/update/pr/body';
```

Comment:
this will cause circular references , so those functions need to be moved elsewhere